zebra-rpc/src/methods.rs

```diff
@@ -526,6 +549,52 @@ where
         }
         .boxed()
     }
+
+    fn get_address_tx_ids(
+        &self,
+        addresses: Vec<String>,
+        start: u32,
+        end: u32,
+    ) -> BoxFuture<Result<Vec<String>>> {
+        let mut state = self.state.clone();
+        let mut response_transactions = vec![];
+        let start = Height(start);
+        let end = Height(end);
+
+        async move {
+            if start > end {
```

Comment:
I think this is good to have but i am unsure how zcashd handle it. I will check.

Reply:
So, i looked into zcashd and found the `start` and `end` argument are validated by a `getHeightRange` function: https://github.com/zcash/zcash/blob/46190bca236757df4979ff6a6e3d2d2d0a2fa8ce/src/rpc/misc.cpp#L811

I added a `check_height_range` helper function into zebra to deal with this in: https://github.com/ZcashFoundation/zebra/pull/4062/commits/eb802af73c74516b1d156cf87bd6958d2a7ee75c

I also found some smallinconsistency in the zcashd implementation, open a PR for it: https://github.com/zcash/zcash/pull/5851